content/programma.md

```diff
@@ -2,5 +2,37 @@
 title: Programma
 date: 2017-03-23
 publishdate: 2017-03-24
-weight: 2
----
\ No newline at end of file
+draft: true
+menu:
+  main:
+    weight: 2
+---
+
+## Dagprogramma
+Hier vind je het dagprogramma. Ben je een daggast? Dan is het dagprogramma samen met het avondprogramma voor jou van toepassing. Geen daggast? Scroll door naar het avondprogramma. 
```

Comment:
Misschien willen we van "avondprogramma" een linkje maken?

Reply:
Done